docs/running-headscale-container.md

```diff
@@ -105,67 +136,26 @@ Verify `headscale` is available:
 curl http://127.0.0.1:9090/metrics
 ```
 
-6. Create a user ([tailnet](https://tailscale.com/kb/1136/tailnet/)):
-
-```shell
-docker exec headscale \
-  headscale users create myfirstuser
-```
-
-### Register a machine (normal login)
-
-On a client machine, execute the `tailscale` login command:
-
-```shell
-tailscale up --login-server YOUR_HEADSCALE_URL
-```
-
-To register a machine when running `headscale` in a container, take the headscale command and pass it to the container:
-
-```shell
-docker exec headscale \
-  headscale --user myfirstuser nodes register --key <YOU_+MACHINE_KEY>
-```
-
-### Register machine using a pre authenticated key
-
-Generate a key using the command line:
-
-```shell
-docker exec headscale \
-  headscale --user myfirstuser preauthkeys create --reusable --expiration 24h
-```
-
-This will return a pre-authenticated key that can be used to connect a node to `headscale` during the `tailscale` command:
-
-```shell
-tailscale up --login-server <YOUR_HEADSCALE_URL> --authkey <YOUR_AUTH_KEY>
+# 7. Launch command in the container :
+ 
+```shell    
+docker exec headscale <COMMAND>
 ```
+    
+!!! note "Debugging & using a shell with Headscale"
 
-## Debugging headscale running in Docker
-
-The `headscale/headscale` Docker container is based on a "distroless" image that does not contain a shell or any other debug tools. If you need to debug your application running in the Docker container, you can use the `-debug` variant, for example `headscale/headscale:x.x.x-debug`.
-
-### Running the debug Docker container
-
-To run the debug Docker container, use the exact same commands as above, but replace `headscale/headscale:x.x.x` with `headscale/headscale:x.x.x-debug` (`x.x.x` is the version of headscale). The two containers are compatible with each other, so you can alternate between them.
-
-### Executing commands in the debug container
+    The `headscale/headscale` Docker container does not contain a shell or any other debug tools. 
+    If you need to debug your application running in the Docker container, you can use the 
+    `-debug` variant, for example `headscale/headscale:0.22.1-debug`.
```

Comment:
Some information is lost like it is a distroless image and that is just a busybox shell, because that can be crucial information in my opinion.

Reply:
Every guys familiar with docker will understand it and the new to docker will just use the debug doc below, adding to much details can lead us to a pain reading the documentation. What's your opinion on that ? I think that docker needs to be an easy way to try Headscale, so we want it as simple as possible.

Reply:
Fair point.